### ISSUE TITLE
pipenv install creates and installs in a new virtualenv when another one is activated)

### ISSUE THREAD

**Issue by glenfant** [pypa/pipenv#1497]
Try this:
```shell
cd ~/project
pipenv install foo
pipenv shell
# ... do some work with virtualenv activated ...
cd ~/sandbox
pipenv install bar
```
This creates a new virtualenv, installs bar and creates `Pipfile` and `Pipfile.lock` in `~/sandbox/` when I expected it to be in the activated virtualenv and `bar` to be included in `~/project/Pipfile` and `~/project/Pipfile.lock`.

As far as I can understand I dunno if it's a bug or a feature. But this seems strange to me. Thanks for pipenv !

**Comment by kennethreitz** [comment]
feature ;)

**Comment by glenfant** [comment]
Okay, this means that we must take care to cd somewhere in the project dir (that has Pipfile) or one of its descendants.

**Comment by techalchemy** [comment]
In case someone searches -- to elaborate slightly -- if `pipenv install` is called inside a pipenv subshell (what you called an activated virtualenv) but outside of a project directory, it will essentially set `PIPENV_IGNORE_VIRTUALENVS` temporarily and create a new environment and a new project

**Comment by pi-bjl** [comment]
Would it really be intended use that you have a virtualenv activated, run 'pipenv install' and did _not_ want the existing virtualenv to be used?

If so, can I suggest to put a warning output message to the effect 'Pipfile not found; creating a new virtualenv' ?

I found it confusing that pipenv did not use the virtualenv I had activated, with pipenv, and instead made a new one and didn't activate it. This 'feature' makes it harder to use pipenv on an existing project with an "idiosyncratic", shall we say, repo layout.

![pipenv](https://user-images.githubusercontent.com/24402833/37575331-bc871670-2b7a-11e8-9c87-18c50ca9055e.png)
